cucumber/authenticators/features/authn_oidc.feature

```diff
@@ -91,31 +96,61 @@ Feature: Users can authneticate with OIDC authenticator
     """
     And I get authorization code for username "bob" and password "bob"
     And I fetch an ID Token
+    And I save the amount of log lines into "bookmark_bob"
     When I authenticate via OIDC with id token
     Then it is unauthorized
+    And The log filtered from line number in "bookmark_bob" should contains "1" messages:
+    """
+    [OIDC] User 'bob' is not authorized to authenticate with webservice 'cucumber:webservice:conjur/authn-oidc/keycloak'
+    """
 
   Scenario: ID token without value of variable id-token-user-property is denied
     When I add the secret value "non_existing_field" to the resource "cucumber:variable:conjur/authn-oidc/keycloak/id-token-user-property"
     And I get authorization code for username "alice" and password "alice"
     And I fetch an ID Token
+    And I save the amount of log lines into "bookmark_alice"
     When I authenticate via OIDC with id token
     Then it is unauthorized
+    And The log filtered from line number in "bookmark_alice" should contains "1" messages:
+    """
+    Authentication Error: #<Authentication::AuthnOidc::IdTokenFieldNotFoundOrEmpty: Field 'non_existing_field' not found or empty in ID Token
+    """
 
   Scenario: Empty or missing id token is a bad request
+    Given I save the amount of log lines into "bookmark_bad_req"
     When I authenticate via OIDC with no id token
     Then it is a bad request
+    And The log filtered from line number in "bookmark_bad_req" should contains "1" messages:
+    """
+    Authentication Error: #<Authentication::MissingRequestParam: field 'id_token' is missing or empty in request body
+    """
 
+    Given I save the amount of log lines into "bookmark_empty_token"
     When I authenticate via OIDC with empty id token
     Then it is a bad request
+    And The log filtered from line number in "bookmark_empty_token" should contains "1" messages:
+    """
+    Authentication Error: #<Authentication::MissingRequestParam: field 'id_token' is missing or empty in request body
+    """
 
   Scenario: non-existing account in request is denied
     Given I get authorization code for username "alice" and password "alice"
     And I fetch an ID Token
+    And I save the amount of log lines into "bookmark_not_existing_acnt"
     When I authenticate via OIDC with id token and account "non-existing"
     Then it is unauthorized
+    And The log filtered from line number in "bookmark_not_existing_acnt" should contains "1" messages:
+    """
+    Authentication Error: #<Conjur::RequiredResourceMissing: Missing required resource: non-existing:variable:conjur/authn-oidc/keycloak/provider-uri
```

Comment:
this will change in GA to "account does not exists" (or something like that) so we should keep in mind this test will crash once we merge

Reply:
added comment to test